src/api.js

```diff
@@ -277,8 +277,10 @@ function qmlweb_parse($TEXT, document_type, exigent_mode) {
       }
     } else if (S.token.type == "name") {
       if (S.token.value == "property" && (S.token.type == "name" || S.token.value == "var")) {
-        next();
-        return qmlpropdef();
+        if (!is_token(peek(), "punc", ":")) {
```

Comment:
Btw, `S.token.value == "property" && (S.token.type == "name" || S.token.value == "var")` could be reduced to `S.token.value == "property" && S.token.type == "name"`, perhaps this means that there is some other bug somewhere.